src/prompts/index.tsx

```diff
@@ -213,18 +218,12 @@ export class PromptManager {
     private async onEnter(state: PromptManagerState) {
         switch (state) {
             case 'loading':
-                this.pushOpsManager = await getPushOpsManager(this.context);
-                await this.pushOpsManager.handleAutoResubscription(
-                    this.context
-                );
-                this.subscriptionState = await this.pushOpsManager.getCurrentSubscriptionState(
```

Comment:
The next line is setting the state to ready which immediately fetches the current subscription state (was called twice). 

Note: the next effort would be to make this whole thing work by subscribing to the push state instead of actively fetching it